src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.labelSearch {
+  box-sizing: border-box;
+  display: block;
+  margin-top: 20px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.labelSearch:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.labelSearch:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.labelSearch-big {
+  height: 70px;
+}
+
+.labelSearch-small {
+  height: 42px;
+}
+
+.inputSearch {
+  background: url(./images/search.svg);
+  background-repeat: no-repeat;
+  border: 0;
+  font-family: Avenir-Book, sans-serif;
+}
+
+.inputSearch-big {
+  background-size: 19px 19px;
+  background-position: 25px 24px;
+  font-size: 16px;
+  padding-left: 62px;
+  padding-top: 23px;
+}
+
+.inputSearch-small {
+  background-size: 11px 11px;
+  background-position: 12px 14px;
+  font-size: 14px;
+  padding-left: 33px;
+  padding-top: 10px;
+}
+
+.inputSearch::placeholder {
+  font-family: Avenir-Book, sans-serif;
```

Comment:
you can move `font-family` to `body` selector and use `font-family: inherit` everywhere